lib/main.dart

```diff
@@ -1,119 +1,98 @@
 import 'package:flutter/material.dart';
+
+import 'package:coffeecard/base/style/colors.dart';
 import 'package:coffeecard/base/style/text_styles.dart';
 
+import 'package:coffeecard/widgets/pages/tickets_page.dart';
+import 'package:coffeecard/widgets/pages/receipts_page.dart';
+import 'package:coffeecard/widgets/pages/stats_page.dart';
+import 'package:coffeecard/widgets/pages/settings_page.dart';
+
 void main() {
   runApp(MyApp());
 }
 
 class MyApp extends StatelessWidget {
-  // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Flutter Demo',
+      title: 'Café Analog',
       theme: ThemeData(
-        // This is the theme of your application.
-        //
-        // Try running your application with "flutter run". You'll see the
-        // application has a blue toolbar. Then, without quitting the app, try
-        // changing the primarySwatch below to Colors.green and then invoke
-        // "hot reload" (press "r" in the console where you ran "flutter run",
-        // or simply save your changes to "hot reload" in a Flutter IDE).
-        // Notice that the counter didn't reset back to zero; the application
-        // is not restarted.
-        primarySwatch: Colors.blue,
-        // This makes the visual density adapt to the platform that you run
-        // the app on. For desktop platforms, the controls will be smaller and
-        // closer together (more dense) than on mobile platforms.
+        primarySwatch: AppColor.createMaterialColor(AppColor.primary),
+        primaryColor: AppColor.primary,
+        brightness: Brightness.dark,
+        backgroundColor: AppColor.background,
         visualDensity: VisualDensity.adaptivePlatformDensity,
       ),
```

Comment:
I suggest this constructor is moved to a static method in a new class in the styles package, to separate concerns and not clutter up the Main class.

Reply:
ThemeData moved to lib/base/style/theme.dart as analogTheme.